src/test/java/com/google/api/generator/gapic/composer/grpc/GrpcServiceStubClassComposerTest.java

```diff
@@ -29,6 +29,7 @@ public void generateGrpcServiceStubClass_simple() {
 
     Assert.assertGoldenClass(this.getClass(), clazz, "GrpcEchoStub.golden");
     Assert.assertEmptySamples(clazz.samples());
+    Assert.assertCodeEquals(clazz.defaultHost(), "");
```

Comment:
I don't think we need the assertions in this tests, because we didn't touch `GrpcServiceStubClassComposer` or `AbstractTransportServiceStubClassComposer`.